### PR TITLE
add tape as last resort for admix download

### PR DESCRIPTION
**Pull request by JoranAngevaare**
Just an idea to make admix download a little easier. For users it's a bit strange if they cannot download while the data is already in Rucio. Although downloading from tape has it's downsides, it's still easier than having to manually check where it's stored and adding a flag to ``admix download --rse ...``.

**EDIT:**
By accident I also looked at travis. It turned out to be slightly different than I anticipated. E.g. the code is not python 2.7 compatible (was tested against) and there are deprecated placeholders e.g. for pypi and the tests don't seem in line with the current state of admix.

### CODE REVIEW

**Comment by ershockley** [comment]
Thanks @jorana, yes Travis for admix was a huge mess -- I guess it was set up by Boris some time ago and never updated.  Thanks for going through that/ 

Are all the changes to `rucio_api` and `rucio_summoner` to satisfy Travis? Or some other motivation? I'm not usually a fan of adding imports inside functions like this.

File: .travis.yml [hunk 1]
```diff
@@ -3,27 +3,27 @@
...
-  provider: pypi
-  distributions: sdist bdist_wheel
-  user: XeBoris
-  password:
-    secure: PLEASE_REPLACE_ME
-  on:
-    tags: true
-    repo: XeBoris/admix
-    python: 2.7
+#deploy:
```
Comment (ershockley) [inline .travis.yml]: Can we just delete instead of commenting?

File: admix/interfaces/rucio_api.py [hunk 2]
```diff
@@ -30,6 +18,21 @@
 import os
 import json
 
+def load_rucio():
```
Comment (ershockley) [inline admix/interfaces/rucio_api.py]: I made comment in main discussion, but can you explain the motivation for this?

File: tests/test_admix.py [hunk 3]
```diff
@@ -1,34 +1,38 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-
+import warnings
 """Tests for `admix` package."""
 
+# This test seems desperatly out of date
```
Comment (ershockley) [inline tests/test_admix.py]: Again i would just remove if they weren't working -- i never used them at least

**Review by ershockley — COMMENTED** [review]
I like the changes to admix-download, but i think we might as well clean up the comments in the tests etc while we're at it. Also we should test that it doesn't affect the uploads, etc (I don't think it will).

File: tests/test_admix.py (hunk 3 above)
Comment (JoranAngevaare) [reply tests/test_admix.py]: Sure sounds good, just wanted to have you on board before deleting other peoples work

**Comment by JoranAngevaare** [comment]
@ershockley , e77dd0e addressed your comments. It's just `from admix.interfaces.rucio_api import RucioAPI` that is still causing issues. I also don't like the way this is imported but moving it somewhere else causes this error on import admix:
```python
(strax) joran@DESKTOP-F4PI41P:~/google_drive/ubuntu-windows/software/admix$ pytest
============================================================================================================ test session starts ============================================================================================================
platform linux -- Python 3.8.5, pytest-6.1.1, py-1.9.0, pluggy-0.13.1
rootdir: /mnt/d/Google_Drive/PhD-master/ubuntu-storage/ubuntu-windows/software/admix
plugins: hypothesis-5.37.1, cov-2.10.1
collected 0 items / 1 error

================================================================================================================== ERRORS ===================================================================================================================
___________________________________________________________________________________________________ ERROR collecting tests/test_admix.py ____________________________________________________________________________________________________
tests/test_admix.py:6: in <module>
    import admix
admix/__init__.py:11: in <module>
    from admix.interfaces.rucio_summoner import RucioSummoner
admix/interfaces/rucio_summoner.py:18: in <module>
    from admix.interfaces.rucio_api import RucioAPI
admix/interfaces/rucio_api.py:21: in <module>
    from rucio.client.client import Client
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/client/__init__.py:20: in <module>
    from .client import Client  # noqa: F401
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/client/client.py:30: in <module>
    from rucio.client.accountclient import AccountClient
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/client/accountclient.py:39: in <module>
    from rucio.client.baseclient import BaseClient
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/client/baseclient.py:92: in <module>
    LOG = setup_logger(module_name=__name__)
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/common/utils.py:1369: in setup_logger
    if _force_cfg_log_level(cfg_opt):
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/common/utils.py:1345: in _force_cfg_log_level
    cfg_forced_modules = config_get('logging', cfg_option, raise_exception=False, default=None, clean_cached=True)
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/common/config.py:53: in config_get
    return get_config().get(section, option)
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/common/config.py:214: in get_config
    __CONFIG = Config()
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/common/config.py:238: in __init__
    raise RuntimeError('Could not load Rucio configuration file. '
E   RuntimeError: Could not load Rucio configuration file. Rucio looked in the following paths for a configuration file, in order:
E       /opt/rucio/etc/rucio.cfg
============================================================================================================= warnings summary ==============================================================================================================
../../../../../../../../home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/client/baseclient.py:45
  /home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/client/baseclient.py:45: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import imp

-- Docs: https://docs.pytest.org/en/stable/warnings.html
========================================================================================================== short test summary info ==========================================================================================================
ERROR tests/test_admix.py - RuntimeError: Could not load Rucio configuration file. Rucio looked in the following paths for a configuration file, in order:
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
======================================================================================================== 1 warning, 1 error in 0.56s ========================================================================================================
```

File: .travis.yml (hunk 1 above)
Comment (JoranAngevaare) [reply .travis.yml]: agree, done in e77dd0e

File: admix/interfaces/rucio_api.py (hunk 2 above)
Comment (JoranAngevaare) [reply admix/interfaces/rucio_api.py]: agree, done in e77dd0e, it wasn't linear getting travis working, was a bit too rigorous here

**Pull request merged**